src/ru/skypro/IntegerListImpl.java

```diff
@@ -0,0 +1,203 @@
+package ru.skypro;
+
+import ru.skypro.exceptions.ElementNotFoundException;
+import ru.skypro.exceptions.InputNullException;
+import ru.skypro.exceptions.WrongIndexException;
+
+import java.util.Arrays;
+
+public class IntegerListImpl implements List{
+
+    private Integer[] storage;
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size;
```

Comment:
константы следует объявлять до обычных полей
```suggestion
    private static final int DEFAULT_CAPACITY = 10;
    
    private Integer[] storage;
    private int size;
```